.env.example

```diff
@@ -1,4 +1,9 @@
 OPENAI_API_KEY=
 TAVILY_API_KEY=
 LANGCHAIN_API_KEY=
-DOC_PATH=./my-docs
\ No newline at end of file
+DOC_PATH=./my-docs
+
+# the name of the embedding model to use for Ollama
+OLLAMA_EMBEDDING_MODEL=
```

Comment:
Don't we need to specify the LLM as well?

Reply:
yes that's correct but there I'm using the environment variables which were already in place. But you are right, they are missing in the `.env.example`:

```bash
LLM_PROVIDER=ollama
EMBEDDING_PROVIDER=ollama
FAST_LLM_MODEL=llama3
SMART_LLM_MODEL=llama3 
```

There is no documentation on the environment variables I think, I can create another PR to document them, maybe in the main `README.md` ?

Reply:
Indeed, it may be good to include all of them in the example. Let us see what @assafelovic says about this before we act on it.

Reply:
@assafelovic no problem, thanks a lot for this amazing project!

Have added the environment variables for the configuration of Ollama to documentation page you have created. I kept it simple and compact just as a code block. 

Have also added the change from the Groq PR (@dphiggs01) regarding the refactoring in llm.py. So there should be no conflict between the two PR.

Reply:
Hey @gschmutz I've merged the Groq support and there are some conflicts. Please go ahead and resolve them and happy to merge this once you approve you've tested and everything works smoothly!